cpp/cpp.rst

```diff
@@ -1,4 +1,5 @@
 :data-transition-duration: 1000
+:data-width: 800
```

Comment:
This is not right. It should be at least 1024. Furthermore, It has to be fixed in the main code of prezenta.